django_lightweight_queue/middleware/transaction.py

```diff
@@ -2,16 +2,13 @@
 
 class TransactionMiddleware(object):
     def process_job(self, job):
-        if not connection.in_atomic_block:
-            transaction.set_autocommit(False)
+        transaction.atomic(savepoint=False).__enter__()
```

Comment:
Seems sensible, we can use savepoints within queue jobs then :). We had some problems yesterday when using savepoint=True I think? Or is that fixed now?